assets/sass/_reset.sass

```diff
@@ -22,6 +22,7 @@ body
   color: color('primary')
   font-family: $text-font
   text-rendering: optimizeLegibility
+  background-color: color('white') 
```

Comment:
It should be `color('default')` according to the change you made in `_variables.sass`?